db/fixtures/miq_product_features.yml

```diff
@@ -339,6 +339,14 @@
       :description: Display Individual Cloud Providers
       :feature_type: view
       :identifier: ems_cloud_show
+    - :name: Arbitration Profiles List
```

Comment:
*Really* not sure here, but feels like renaming product features needs a migration to update the existing users with that feature?